src/index.tsx

```diff
@@ -10,6 +10,7 @@ import {createGlobalStyle} from 'styled-components';
 import 'fontsource-dm-sans';
 import 'fontsource-work-sans';
 import 'fontsource-work-sans/600.css';
+import 'fontsource-work-sans/300.css';
```

Comment:
Is there any reason to use `300` specifically over the default? We should avoid importing too many fonts (we're already adding a monospace font in another PR).

Reply:
For whatever reason if you don't specify the font size it only imports font weight 400... as for whether it counts as a new font that depends on whether all the fonts are bundled into the production build (as installed via yarn) or whether it only bundles the imported ones.

Reply:
Point being, adding a new font _weight_ will increase load times, and if it can be avoided then we should leave it out.

Reply:
Alright. I've replaced all uses of the 300 weight font in figma. We can remove it here @OmarG247.